geoh5py/workspace/workspace.py

```diff
@@ -1064,24 +1097,25 @@ def load_entity(
         uid: uuid.UUID,
         entity_type: str,
         parent: Entity | None = None,
-    ) -> Entity | None:
+    ) -> Entity | PropertyGroup | None:
         """
         Recover an entity from geoh5.
-
         :param uid: Unique identifier of entity
         :param entity_type: One of entity type 'group', 'object', 'data' or 'root'
-
+        :param parent: Parent entity.
         :return entity: Entity loaded from geoh5
         """
-        if isinstance(self.get_entity(uid)[0], Entity):
-            return self.get_entity(uid)[0]
+        get_entity_ = self.get_entity(uid)[0]
+        if isinstance(get_entity_, (Entity, PropertyGroup)):
+            return get_entity_
```

Comment:
```suggestion
        child = self.get_entity(uid)[0]
        if isinstance(child, (Entity, PropertyGroup)):
            return child
```

Not really PEP8 compliant.